docs/mkdocs.yml

```diff
@@ -1,31 +1,49 @@
+# This file manages Artery's website, for more information see https://www.mkdocs.org 
+
 site_name: Artery V2X Simulation Framework
-site_dir: www
+site_dir: ../www
 repo_name: 'Artery on GitHub'
 repo_url: https://github.com/riebl/artery
+docs_dir: website
 
 theme:
-    name: 'material'
+    name: material
     features:
         - tabs
+        - content.code.copy    # Adds "copy to clipboard" button to code blocks
     icon:
         logo: fontawesome/solid/wifi
     palette:
+      # Palette toggle for light mode
+      - media: "(prefers-color-scheme: light)"
+        primary: 'teal'
+        accent: 'deep orange'
+        toggle:
+          icon: material/weather-night
+          name: Switch to dark mode
+      # Palette toggle for dark mode
+      - media: "(prefers-color-scheme: dark)"
+        scheme: slate
         primary: 'teal'
         accent: 'deep orange'
-    font:
-        text: 'Lato'
-        code: 'Oxygen Mono'
```

Comment:
TODO: I messed this up

Reply:
Fixed in [0dd85cb8350b36b1aa7d1727c02fc345fae56a00](https://github.com/riebl/artery/pull/301/commits/0dd85cb8350b36b1aa7d1727c02fc345fae56a00)